includes/html/table/inventory.inc.php

```diff
@@ -61,11 +61,11 @@
     $sql .= " LIMIT $limit_low,$limit_high";
 }
 
-$sql = "SELECT `D`.`device_id` AS `device_id`, `D`.`hostname` AS `hostname`,`entPhysicalDescr` AS `description`, `entPhysicalName` AS `name`, `entPhysicalModelName` AS `model`, `entPhysicalSerialNum` AS `serial` $sql";
+$sql = "SELECT `D`.`device_id` AS `device_id`, `D`.`hostname` AS `hostname`, `D`.`sysName` AS `sysName`,`entPhysicalDescr` AS `description`, `entPhysicalName` AS `name`, `entPhysicalModelName` AS `model`, `entPhysicalSerialNum` AS `serial` $sql";
 
 foreach (dbFetchRows($sql, $param) as $invent) {
     $response[] = array(
-                   'hostname'    => generate_device_link($invent, shortHost($invent['hostname'])),
+                   'hostname'    => generate_device_link($invent),
```

Comment:
to see complete hostname, instead of short variant of hostname you can change length oft shortend hostname for your needs in global settings here:
![image](https://user-images.githubusercontent.com/7978916/74105651-01f50580-4b60-11ea-90d1-0c94b27ab77c.png)

As you see, i extended the default value from 12 to 30 for myself to see complete FQDN.
So i don't see any need of changing code here.

Reply:
This change is probably improving the behaviour with devices that are added by IP and where sysname is necessary. So indeed this one should be kept.

Reply:
Tested it in a ip-hostname environnement. Works as expected.

Reply:
Indeed this is done to see the real hostname (from SNMP) instead of seeing the IP on an environment where all the devices are added by IP.